src/screens/Login/Login/LoginContainer.js

```diff
@@ -67,16 +67,14 @@ class LoginContainer extends PureComponent {
   }
 
   authenticate = () => {
-    return TouchID.authenticate()
+    return TouchID.authenticate('ChronoWallet Application')
```

Comment:
May we use a constant here instead of 'ChronoWallet Application'?

Reply:
Yeah i will change. It was for testing purposes. Should we add it to a constant or add it to i18n?